build/Microsoft.DotNet.Cli.Prepare.targets

```diff
@@ -0,0 +1,55 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="14.0" DefaultTargets="Layout" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  
+  <UsingTask TaskName="Rid" AssemblyFile="$(CLIBuildDll)" />
+  <UsingTask TaskName="Architecture" AssemblyFile="$(CLIBuildDll)" />
+  <UsingTask TaskName="GenerateBuildVersionInfo" AssemblyFile="$(CLIBuildDll)" />
+
+  <Target Name="Init" >
+    <!-- Current Runtime Information -->
+    <Rid>
+      <Output TaskParameter="OutputRid" PropertyName="Rid" />
+    </Rid>
+    <Architecture>
+      <Output TaskParameter="OutputArchitecture" PropertyName="Architecture" />
+    </Architecture>
+
+    <!-- Common Properties -->
+    <PropertyGroup>
+      <RepoRoot>$(MSBuildThisFileDirectory)/..</RepoRoot>
```

Comment:
I think this property should go in build.proj, or even better a dirs.props file in the root.